pom.xml

```diff
@@ -66,13 +66,19 @@
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-api</artifactId>
-        <version>${slf4j.version}</version>
+        <version>[${slf4j.version},1.7.9999)</version>
       </dependency>
       <dependency>
         <groupId>ch.qos.logback</groupId>
         <artifactId>logback-classic</artifactId>
-        <version>${logback.version}</version>
+        <version>[${logback.version},1.2.9999)</version>
```

Comment:
What if a security issue is found in a version of logback that it greater than the one we use ?